generics/services/azure.js

```diff
@@ -102,9 +102,67 @@ let getDownloadableUrl = function (filePath, containerName) {
   });
 }
 
+ /**
+  * Get azure cloud signed url.
+  * @method
+  * @name signedUrl
+  * @param {String} [folderPath = ""] - link to the folder path.
+  * @param {Array} [fileName = ""] - fileName.
+  * @param {Array} containerName - name of the container.  
+  * @return {Object} - signed url and azure file name. 
+*/
+
+let signedUrl = (folderPath = "", fileName = "",containerName) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+      
+      if(folderPath == "" || fileName == "") {
+        throw new Error(httpStatusCode.bad_request.status);
+      }
+      
+      let startDate = new Date();
+      startDate.setMinutes(startDate.getMinutes() - 5);
+      let expiryDate = new Date(startDate);
```

Comment:
@aman-tunerlabs - Set expiry from a common constant across services

Reply:
@aks30  Expiry logic is different for different services. So thats why i did not kept it common